### PR TITLE
fix: Minor docs changes

### PR DESCRIPTION
**Pull request by Aju100**
Hi Opensource team,

As the discussion goes on discord group, I had made PR for it where I removed $ in order to execute commands on the terminal. 

Regards, 
Aju Tamang

### CODE REVIEW

File: docs/source/installation.rst [hunk 1]
```diff
@@ -24,33 +24,33 @@ Installation with PyPi
 
 In a terminal, issue the following command::
 
-    $ pip install fury
+    pip install fury
```
Comment (Nibba2018) [inline docs/source/installation.rst:28]: remove the "::" and replace the command with:

.. code-block:: shell

  pip install fury

File: docs/source/installation.rst [hunk 2]
```diff
@@ -97,14 +97,14 @@ Since Xvfb will require an X server (we also recommend to install XQuartz packag
...
 - First option::
 
-    $ export DISPLAY=:0
-    $ Xvfb :0 -screen 1920x1080x24 > /dev/null 2>1 &
-    $ pytest -svv fury
+    export DISPLAY=:0
+    Xvfb :0 -screen 1920x1080x24 > /dev/null 2>1 &
+    pytest -svv fury
 
 - Second option::
```
Comment (Nibba2018) [inline docs/source/installation.rst:105]: Same as above and for the rest of the shell commands.

**Review by Nibba2018 — COMMENTED** [review]
Hi @Aju100 , Thank you for digging through the documentation and creating this PR. I think it would be better if they are converted to code blocks instead. Code blocks will provide syntax highlighting along with formatting hence serving our purpose.

I have added some comments below do have a look.

File: docs/source/installation.rst (hunk 1 above)
Comment (Aju100) [reply docs/source/installation.rst]: thank you @Nibba2018  got to learn about rst too. I request you to have a review and some feedbacks about it

File: docs/source/installation.rst (hunk 2 above)
Comment (Aju100) [reply docs/source/installation.rst]: I have done as you can review out my latest commit.

**Review by skoudoro — COMMENTED** [review]
Thank you for this @Aju100 and thanks for the review @Nibba2018.

Overall, it looks good to me. I just need to generate this locally and then I will go ahead and merge it. Thank you @Aju100

**Review by skoudoro — APPROVED** [review]

**Comment by Nibba2018** [comment]
Thank you @Aju100 for this PR and Thank you @skoudoro for the review merging! 🚀

**Pull request merged**